contents/handbook/people/compensation.mdx

```diff
@@ -83,18 +83,30 @@ If you are nomading, we will set your location factor for the place that you are
+As part of your compensation, you will receive share options in the company. Broadly, the amount of options will depend on the Level as per the Experience Factor. We may change this policy from time to time depending on our rate of hiring - e.g. if we had a gap in hiring for an extended period, we would adjust this.
 
-Whilst the terms of options for _any company_ could vary if we were ever acquired, we have set them up with the following key terms:
+Whilst the terms of options for _any company_ could vary if we were ever acquired, we have set them up with the following key terms which we believe are industry-leading in their friendliness to employees:
 
+-   Standard 4-year vesting with a 1-year cliff
 -   10 years to exercise your options in the event that you leave PostHog
 -   Double trigger acceleration, which means if you are let go or forced to leave due to the company being acquired, you receive all of your options at that time
 -   Vesting starts from your start date (not after a "probation period" or similar)
 -   For UK-based team members, our options are part of the [EMI share options scheme](https://www.gov.uk/tax-employee-share-schemes/enterprise-management-incentives-emis), which is tax-advantaged
 
-It can take time to approve options, as it requires a board meeting and company valuation. We can clarify the likely time frame at the time we're hiring you. In any case, you will not be disadvantaged as vesting will always start from when you joined PostHog. While we can commit to a particular percentage, we cannot commit to a particular strike price when offering share options, as the valuations are done by a third party and can vary depending on where we are in our funding cycle. 
+It can take time to approve options, as it requires a board meeting and company valuation. We can clarify the likely time frame at the time we're hiring you. Vesting will always start from when you joined PostHog, not from when you receive your option agreement. While we can commit to a particular percentage, we cannot commit to a particular strike price when offering share options, as the valuations are done by a third party and can vary depending on where we are in our funding cycle.
 
+Check out our [share options FAQs](/handbook/people/share-options#frequently-asked-questions) to learn more. 
 
+### Trading cash compensation for equity
+
+As the value of PostHog equity has risen significantly in a short period of time, we want to give everyone the option to trade any future pay rises for more share options. We also want to give everyone more flexiblity in their overall compensation.  
+
+- You can trade up to 100% of any pay rise you receive for share options at a 1:1 ratio of cash to equity
```

Comment:
Just to clarify, does this mean if I get a raise of, say, $5,000, and the current strike price is $5 per share, then I can choose to get either:

1. $5000 / year more every year.
2. 1,000 shares that vest over 4 years? (or is this 4,000 shares over 4 years?)

Reply:
If you got a pay rise of $5k cash today, you could choose to instead receive share options equal to $5k/year (ie. $20k that vests over 4 years). 

So if today's share price was $5 per share, you would receive 4,000 shares that vest over 4 years - though I would focus on the value of shares rather than the number because the latter depends on when we allocate them (which I think is driving your second question below).

Reply:
1. Would these shares have a 1-year cliff too or just vest monthly? As if you opted for money, you'd receive it every month.
2. Can you opt to do partial (e.g. 50% cash raise, 50% options)?

Reply:
1. Yeah to make this more attractive in future we'd probably want to change the vesting schedule to monthly. However to keep it simple and reduce additional admin work, we're following our existing share option plan (4 year vesting, 1 year cliff). If you received money it would also lose 2-3% of its value every year depending on inflation where you live 😉. 
2. Yep, anything from 0-100%.